Sources/ArcGISToolkit/Components/Form/FormInputs/MultiLineTextInput.swift

```diff
@@ -63,36 +63,11 @@ struct MultiLineTextInput: View {
     var body: some View {
         InputHeader(label: element.label, isRequired: inputModel.isRequired)
             .padding([.top], elementPadding)
-        HStack(alignment: .bottom) {
-            if #available(iOS 16.0, *) {
-                TextEditor(text: $text)
-                    .scrollContentBackground(.hidden)
-                    .disabled(!inputModel.isEditable)
-            } else {
-                TextEditor(text: $text)
-                    .disabled(!inputModel.isEditable)
-            }
-            if isFocused && !text.isEmpty && inputModel.isEditable {
-                ClearButton { text.removeAll() }
-            }
+        if inputModel.isEditable {
+            textEditor
+        } else {
+            Text(text.isEmpty ? "--" : text)
```

Comment:
Can you make sure Kotlin is doing this as well?

Reply:
They will open an issue to add it as discussed in #apollo just now.